content/articles/modrictin-include-vite-built-js-and-css-files-in-your-project.md

```diff
@@ -0,0 +1,85 @@
+---
+name: Have multiple Vite and Tailwind bundles in your Filament project
+slug: modrictin-include-vite-built-js-and-css-files-in-your-project
+author_slug: modrictin
+publish_date: 2023-08-17
+categories: [ general,tailwind-css,panel-builder ]
+type_slug: trick
+---
+
+## The problem
+
+What if you want to use the Vite build files in your Filament project?
+You also want to utilize the HMR feature of Vite. How do you do that?
+
+## The solution
+
+You can use Vite facade to include the Vite build files in your Filament project.
+
+In general, you can use the `Illuminate\Support\Facades\Vite` facade to include the Vite build files in your project.
+
+### Get asset url
+
+Please note, the second parameter in the `asset()` function is the build directory.
+
+```php
+Vite::useHotFile('admin.hot')
```

Comment:
Please fix code example indentation across all code examples in your articles

Reply:
I think it should be good now, thanks!